platform/linux-generic/arch/default/odp/api/abi/hash_crc32.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2021, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#ifndef ODP_API_ABI_HASH_CRC32_H_
+#define ODP_API_ABI_HASH_CRC32_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+
+uint32_t _odp_hash_crc32_generic(const void *data, uint32_t data_len,
+				 uint32_t init_val);
+uint32_t _odp_hash_crc32c_generic(const void *data, uint32_t data_len,
+				  uint32_t init_val);
+
+static inline uint32_t _odp_hash_crc32(const void *data, uint32_t data_len,
+				       uint32_t init_val)
+{
+	return _odp_hash_crc32_generic(data, data_len, init_val);
+}
+
+static inline uint32_t _odp_hash_crc32c(const void *data, uint32_t data_len,
+					uint32_t init_val)
+{
+	return _odp_hash_crc32c_generic(data, data_len, init_val);
+}
```

Comment:
Is it a problem that these calls are not inlined? So even though we have hash_inlines.h in this commit, it only inlines one call level and the actual implementation is not inlined.

Reply:
The reasoning for this solution was that it doesn't make the current situation worse (assuming abi-compat is disabled). Proper inlining for the generic functions can be implemented in separate PR.

Reply:
OK, that sounds reasonable.